src/browser/components/atoms/Userimage/userImage.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+import styles from './userimage.css'
```

Comment:
where are you using the `styles` variable? If not just do `import  './userimage.css'`